esmvaltool/cmorizers/obs/cmor_config/GRACE.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Global attributes of NetCDF file
+attributes:
+  dataset_id: GRACE
+  project_id: OBS6
+  tier: 3
+  version: '1'
+  modeling_realm: satellite
+  source: ''
+  reference: 'grace'
+  comment: |
+    'Grace'
+
+# Variables to CMORize
+variables:
+  lweGrace:
+    mip: Lmon
+    raw: lwe_thickness
+    file: 'GRCTellus.JPL.200204_202003.GLO.RL06M.MSCNv02CRI.nc'
+
```

Comment:
The file name is not correct anymore as the dataset was updated with 2020 data. The previous file is not available anymore. The correct name is: GRCTellus.JPL.200204_202009.GLO.RL06M.MSCNv02CRI.nc